lib/chatops/chatops.go

```diff
@@ -45,6 +45,11 @@ func (c *ChatOps) PostMessage(code int, target string) (*ChatOpsResponse, error)
 		}
 		resp.Channel = res.Channel
 		resp.Timestamp = strconv.FormatInt(res.Ts, 10)
+	case "teams":
+		_, err := c.teams(code, target, output)
```

Comment:
What is this value that is not being set to a variable? What is the reasoning behind not setting it?

Reply:
That value is the adaptiveCard, which is being sent to the Teams webhook. I'm trying to create a test for the teams() function, but couldn't figure out to parse the JSON in Go yet.

In the PostMessage() it's of no use, because that function returns a ChatOpsResponse struct with Timestamp and Channel fields. But those values are not returned from the Teams webhook response.

I may remove the return value, until the test is finished.